### ISSUE TITLE
Use slug instead of Id

### ISSUE THREAD

**Issue by Nuhel** [outer-web/filament-link-picker#1]
I see its storing id as parameter, but i want to store slug. How can i achieve this?

This is the stored value.
{"route_name":"event.show","parameters":{"event":"1"},"options":{"opens_in_new_tab":false}}

```php

Route::get('/event/{event:slug}', 'show')->name('show')->filamentLinkPicker(
        label: 'Single Event',
        group: 'Details Page',
        parameterLabels: [
            'slug' => 'event:slug',
        ]
    );
```

**Comment by SimonBroekaert** [comment]
Amazing feature! Thank you so much @Nuhel !

**Comment by SimonBroekaert** [comment]
@Nuhel ,

This is now available in v3.0.0!

Thanks again!
